md/ui-designer-overview.md

```diff
@@ -57,3 +57,24 @@ To use a layout in a business application, export it from the Bonita UI designer
 
 The UI Designer is intended for web developers creating pages and forms that will be part of a BPM application. It requires a good understanding of JavaScript and CSS.  
 If you are a business analyst, you can use the UI Designer to create the page structure and define basic behaviors. However, most properties of a page are defined using JavaScript.
+
+## UI Designer with collaboration
+
+Id of Pages or forms who are created in 7.6 older version is an uuid.
+
+With 7.7 version and the improvement of collaboration, the Id of created artifact is now more human readable and understandable. This Id is now equals of the name.
```

Comment:
and the improvements made for the collaboration features, the Id of the new pages is now...